docs/concepts/key-terms/tracing/trace-view.mdx

```diff
@@ -48,6 +48,15 @@ By default, the waterfall view shows a list of transactions. If you need more gr
 </div>
 
 <br />
+
+### Previous and Next Traces
+
+As users interact with the application, they generate multiple traces. For example, when a user loads a page in their browser and then navigates to another page, this creates two separate traces: one for the initial pageload and another for the subsequent navigation.
```

Comment:
Since the second paragraph starts with "linked traces", I think we need to introduce this a bit earlier 

(feel free to go with your own version as you see fit)

```suggestion
As users interact with the application, they generate multiple traces. For example, when a user loads a page in their browser and then navigates to another page, this creates two separate traces: one for the initial pageload and another for the subsequent navigation. Sentry will link these traces for you.
```